.github/ISSUE_TEMPLATE/feature-enhancement.yml

```diff
@@ -0,0 +1,53 @@
+name: "📈 Enhance existing function"
```

Comment:
We have them together since a lot of times it is miscategorized.  Someone may want a generic feature but think it should be inside FancyZones for instance

Reply:
I agree with @crutkas. How should a user knows every time if it's an enhancement or a new feature?!